test/system/220-healthcheck.bats

```diff
@@ -273,4 +273,215 @@ Log[-1].Output   | \"Uh-oh on stdout!\\\nUh-oh on stderr!\\\n\"
     done
 }
 
+@test "podman healthcheck --health-max-log-count default value (5)" {
```

Comment:
It seem there is rather significant duplication in what is being tested in e2e and system tests. We should not have duplicate tests as this just wastes CI time. One of them is good enough. I think using system tests make sense here as they run in gating.

And it is extremely difficult for me to figure out if these tests are indeed the same or if there are some minor differences.

Reply:
The tests test the same functionality. I will remove the e2e tests.